File: Leet_Code/Remove_Duplicates_from_Sorted_Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        if len(nums) == 0:
            return 0;
        i = 0;
        for k in range(1,len(nums)):
            if nums[k] != nums[i]:
                i += 1
                nums[i] = nums[k]     
        return i+1

File: Leet_Code/Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(!size(nums)){
            return 0;
        }
        int i = 0;
        for(int k = 1; k < size(nums); k++){
            if(nums[i] != nums[k]){
                nums[++i] = nums[k];
            }
        }
        return i+1;
            
        
    }
};